qiskit-ibm-runtime-mcp-server/src/qiskit_ibm_runtime_mcp_server/ibm_runtime.py

```diff
@@ -753,4 +753,130 @@ async def get_service_status() -> str:
         return f"IBM Quantum Service Status: {status_info}"
 
 
+@with_sync
+async def delete_saved_account(account_name: str = "") -> str:
+    """Delete a saved account.
+    Args:
+        account_name: Name of the saved account to delete.
+
+    Returns:
+    Information about if an account was deleted.
+    """
```

Comment:
All the new functions have too brief descriptions compared to the previous ones. 

Add comprehensive docstrings describing what each tool does, its arguments, return values, and any side effects (especially for `delete_saved_account` which is destructive).